x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/configured_sources_list.tsx

```diff
@@ -107,7 +109,11 @@ export const ConfiguredSourcesList: React.FC<ConfiguredSourcesProps> = ({
                       responsive={false}
                     >
                       <EuiFlexItem>
-                        <SourceIcon serviceType={serviceType} name={name} size="xxl" />
+                        <SourceIcon
+                          serviceType={baseServiceType || serviceType}
```

Comment:
question: Can we use `??` instead of `||` as a shortcut here? It should be slightly more accurate because it treats an empty string as truthy.

Reply:
[Done!](https://github.com/elastic/kibana/pull/131802/commits/2eb0527d75ac5b330994ebb150d03322295871a5#diff-2b02d4f4ed0e4a4304a28d1aece3e287dd811895dbee64cd6d0c84426c126c3eR113)